playgrounds/astro/package.json

```diff
@@ -0,0 +1,27 @@
+{
+  "name": "astro-playground",
+  "type": "module",
+  "version": "0.0.1",
+  "private": true,
+  "scripts": {
+    "dev": "astro dev",
+    "start": "astro dev",
+    "build": "astro build",
+    "preview": "astro preview",
+    "astro": "astro",
+    "clean": "rimraf .turbo && rimraf node_modules && rimraf dist"
```

Comment:
I'm not sure when is `.turbo` generated, but after running `yarn dev`, `.turbo` is not generated in `/playgrounds/astro`. So just confirming if this is needed.

Also, is `dist` some type of output folder from turbo?

Reply:
`dist` is the output from astro. `.turbo` is where turbo puts any cached stuff and i think logs.